src/Explorer/Notebook/useNotebook.ts

```diff
@@ -312,8 +312,8 @@ export const useNotebook: UseStore<NotebookState> = create((set, get) => ({
 
       if (dbAccountAllowedInfo.status === HttpStatusCodes.OK) {
         if (dbAccountAllowedInfo?.type === PhoenixErrorType.PhoenixFlightFallback) {
-          isPhoenixNotebooks = isPublicInternetAllowed && userContext.features.phoenixNotebooks;
-          isPhoenixFeatures = isPublicInternetAllowed && userContext.features.phoenixFeatures;
+          isPhoenixNotebooks = isPublicInternetAllowed && userContext.features.phoenixNotebooks === true;
+          isPhoenixFeatures = isPublicInternetAllowed && userContext.features.phoenixFeatures === true;
```

Comment:
It's been a while since I've done some typescript, Is this additional check to make sure that phoenixNotebooks and phoenixFeatures is assigned a bool and not some other value that would also return true when cast to a bool?

Reply:
Yes, in case of "userContext.features.phoenixNotebooks" undefined, "isPhoenixNotebooks" also becomes undefined. This will eliminate the case.